curl/CMakeLists.txt

```diff
@@ -13,4 +19,8 @@ endif()
 add_leatherman_library(src/client.cc src/request.cc src/response.cc)
 add_leatherman_headers(inc/leatherman)
 
-target_link_libraries(leatherman_curl "${CURL_LIBRARIES}")
+if(${WIN32})
+    add_definitions(-DCURL_STATICLIB)
```

Comment:
We don't want to do this for everything, just tests. I lean towards defining it in the test files before including `curl/curl.h`.